EIPS/eip-6049.md

```diff
@@ -0,0 +1,41 @@
+---
+eip: 6049
+title: Deprecate SELFDESTRUCT
+description: Deprecate SELFDESTRUCT by discouraging its use and warning about a potential future behavior change.
+author: William Entriken (@fulldecent)
+discussions-to: https://ethereum-magicians.org/t/deprecate-selfdestruct/11907
+status: Draft
+type: Standards Track
+category: Core
+created: 2022-11-27
+---
+
+## Abstract
+
+This EIP deprecates the `SELFDESTRUCT` opcode and warns against its use. A breaking change to this functionality is likely to come in the future.
+
+## Motivation
+
+Discussions about how to change `SELFDESTRUCT` are ongoing. But there is a strong desire to do *something*.
+
+## Specification
+
+Documentation of the `SELFDESTRUCT` opcode is updated to warn against its use and to note that a breaking change may be forthcoming.
+
+## Rationale
+
+As time goes on, the cost of doing something increases, because any change to `SELFDESTRUCT` will be a breaking change.
+
+The Ethereum Blog and other official sources have not provided any warning to developers about a potential forthcoming change.
+
+## Backward Compatibility
+
+This EIP udpates non-normative text in the Yellow Paper. No changes to clients is applicable.
+
+## Security Considerations
+
+None.
```

Comment:
![image](https://user-images.githubusercontent.com/45835846/204163587-9eeb4f53-a41c-421a-9782-159427887a9e.png)

Reply:
Never mind. This EIP does less than what I thought it did.

Reply:
:-)